### PR TITLE
px4iofirmware: ignore safety when override is enabled

### PR DESCRIPTION
**Pull request by tridge**
this forces PWM outputs in override, ignoring the safety state. This
makes in-flight reboot safer. ArduPilot has had this behaviour for a
while, but it was done in the FMU code by overriding the safety
state. That was not ideal as it didn't work during bootup and was racy
at all times.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Jenkins test this please.

File: src/modules/px4iofirmware/mixer.cpp [hunk 1]
```diff
@@ -167,9 +180,10 @@ mixer_tick(void)
 	 * FMU or from the mixer.
 	 *
 	 */
+	bool effective_safety_off = (r_status_flags & PX4IO_P_STATUS_FLAGS_SAFETY_OFF) != 0 || override_enabled;
```
Comment (LorenzMeier) [inline src/modules/px4iofirmware/mixer.cpp]: This would lead on PX4 to immediate arming in manual flight mode. I need to find something that does only bypass arming on reboots.

Comment (julianoes) [inline src/modules/px4iofirmware/mixer.cpp]: @tridge How does the in-air restart work for APM? Are the PWM kept at the last value until the FMU comes back, or are they switched off?
I'm trying to understand why this change is necessary.
